ledger-core/v2/application/preprocessor/templates/proxy.go.tpl

```diff
@@ -20,6 +20,7 @@
 package {{ .PackageName }}
 
 import (
+	"github.com/insolar/assured-ledger/ledger-core/v2/insolar/payload"
```

Comment:
we do all imports [in the code](https://github.com/insolar/assured-ledger/blob/08a8da167bc74a667002ec639ed59c9b4e105030/ledger-core/v2/application/preprocessor/main.go#L609)

Reply:
thats not entirely true, https://github.com/insolar/assured-ledger/blob/08a8da167bc74a667002ec639ed59c9b4e105030/ledger-core/v2/application/preprocessor/templates/initialization.go.tpl#L23
I didn't want to touch this generateImport function, because off poor design with wrapper/notwrapper signature. I'll think what could be done, looks ugly anyway

Reply:
poor design is a poor excuse. Consistency wins while we're looking for beautiful looking way.

Reply:
ok, now all imports are in generator, nothing lives in templates to avoid confusions in futute